tests/test_cosmologger.py

```diff
@@ -203,6 +203,29 @@ def test_human_format_invalid(cosmolog, cosmolog_setup):
     assert logline == 'Apr 13 03:07:53 jupiter.planets.com star_stuff: [ERROR] BadLogFormat("the {blarg} has exploded") {\'component\': \'oxygen tank\'}'  # noqa: E501
 
 
+def test_json_not_bad(cosmolog, cosmolog_setup):
+    logstream = cosmolog_setup(formatter='human')
+    logger = cosmolog()
+    msg = 'somtimes somebody logs a {"json": "data"} from space. That is okay.'
```

Comment:
s/somtimes/sometimes unless you were going for that :) (and same below)

Reply:
:facepalm: fixed